lib/galaxy/web/base/interactive_environments.py

```diff
@@ -69,6 +71,22 @@ def __init__(self, trans, plugin):
         else:
             self.attr.proxy_prefix = ''
 
+    def load_allowed_images(self):
+        if os.path.exists(os.path.join(self.attr.our_config_dir, 'allowed_images.yml')):
+            fn = os.path.join(self.attr.our_config_dir, 'allowed_images.yml')
+        elif os.path.exists(os.path.join(self.attr.our_config_dir, 'allowed_images.yml.sample')):
+            fn = os.path.join(self.attr.our_config_dir, 'allowed_images.yml.sample')
+        else:
+            raise Exception("Could not find allowed_images.yml file for " + self.attr.viz_id)
```

Comment:
It seems to me like this requires an allowed_images.yml file (or sample) - and phinch and iobio don't have one. Can we fallback to just the ``ini`` for these simpler IEs that function more like a single dataset viz (where the history panel button make a lot more sense).